nupic/frameworks/opf/metrics.py

```diff
@@ -1459,7 +1424,62 @@ def addInstance(self, groundTruth, prediction, record = None):
 
     return self.aggregateError
 
+###################################################################################
+class MetricMulti(MetricsIface):
+  """Multi metric can combine multiple other (sub)metrics and 
+     weight them to provide combined score."""
 
+  def __init__(self, metricSpec):
+    """MetricMulti constructor using metricSpec is not allowed."""
+    raise ValueError("MetricMulti cannot be constructed from metricSpec string! "
+                     "Use MetricMulti(weights,metrics) constructor instead.")
+
+  def __init__(self, weights, metrics, id=None, window=None):
+    """MetricMulti 
+       @param weights - [list of floats] used as weights
+       @param (sub)metrics - [list of metrics] 
+       @param id - (opt) name of the metric
```

Comment:
other metrics can use the existing `minimize=<string>` form, multi metric cannot be constructed using this string, thus needs a special case - the id.